src/Footer/PerPage/Per-page.jsx

```diff
@@ -1,19 +1,28 @@
+import React, { useState } from "react";
 import "./Style-Per-page.css";
+export const PerPage = ({ array, render }) => {
 
-function PerPage() {
+    const [selectValue, setselectValue] = useState()
+    let copieArr=[...array].slice(0, selectValue)
+    const selectValueRows = e => {
+        setselectValue(e.target.value)
+        render(copieArr)
+    };
+    
+    console.log(selectValue)
+    //console.log(array)
```

Comment:
WHAT IS THIS?

Reply:
function to select rows perpage but have error